tests/worker/workflow_sandbox/test_restrictions.py

```diff
@@ -16,8 +16,8 @@
 
 
 def test_workflow_sandbox_stdlib_module_names():
-    if sys.version_info != (3, 10):
-        pytest.skip("Test only runs on 3.10")
+    if sys.version_info[1] != 11:
```

Comment:
I was improperly checking it (the tuple is actually like 5 values, so my `!=` wasn't doing the right thing anyways). Since we are never expected to be off version 3 anyways, this was simpler than manually destructuring.